plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3select/S3SelectCsvRecordReader.java

```diff
@@ -91,4 +92,11 @@ public SelectObjectContentRequest buildSelectObjectRequest(Properties schema, St
 
         return selectObjectRequest;
     }
+
+    protected String getFieldDelimiter(Properties schema)
+    {
+        // Use the field delimiter only if it is specified in the schema. If not, send it as null in the request to S3 Select.
+        // In this case, S3 Select defaults to using ',' as the field delimiter.
+        return schema.getProperty(FIELD_DELIM);
```

Comment:
if we were to read the file directly, we would need to assume the default delimiter (i guess it's comma, right?)
why not add the defaulting logic here too, so that it remains on the Trino side, and is not up to an external service?